beacon-chain/rpc/eth/helpers/http.go

```diff
@@ -0,0 +1,41 @@
+package helpers
```

Comment:
Why is this `helpers` package nested inside the `eth` subpackage?

Reply:
moved to `/network`